File: HolaMundoJava/app/src/main/java/com/example/holamundojava/MainActivity.java
package com.example.holamundojava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button buttonCalculadora, buttonGrafico, buttonCrud, buttonSalir, buttonCalculadoraBasica, buttonWS;

    // Método onCreate
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        buttonCalculadora = findViewById(R.id.buttonCalculadora);
        buttonGrafico = findViewById(R.id.buttonGrafico);
        buttonCrud = findViewById(R.id.buttonCrud);
        buttonSalir = findViewById(R.id.buttonSalir);
        buttonCalculadoraBasica = findViewById(R.id.buttonCalculadoraBasica);
        buttonWS = findViewById(R.id.buttonWS);

        buttonCalculadoraBasica.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, CalculadoraBasicaActivity.class);
                startActivity(intent);
            }
        });

        buttonCalculadora.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, CalculadoraActivity.class);
                startActivity(intent);
            }
        });

        buttonGrafico.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, GraficoActivity.class);
                startActivity(intent);
            }
        });

        buttonCrud.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, CrudActivity.class);
                startActivity(intent);
            }
        });

        buttonSalir.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });

        buttonWS.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, WSActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: HolaMundoKotlin/app/src/main/java/com/example/holamundokotlin/MainActivity.kt
package com.example.holamundokotlin
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    /*
    var buttonCalculadora = Button(this)
    var buttonGrafico = Button(this)
    var buttonCrud = Button(this)
    var buttonSalir = Button(this)
    */

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val buttonSalir = findViewById<Button>(R.id.buttonSalir)
        val buttonCalculadora = findViewById<Button>(R.id.buttonCalculadora)
        val buttonGrafico = findViewById<Button>(R.id.buttonGrafico)
        val buttonCrud = findViewById<Button>(R.id.buttonCrud)
        val buttonCalculadoraBasica = findViewById<Button>(R.id.buttonCalculadoraBasica)
        val buttonWSTienda = findViewById<Button>(R.id.buttonWSTienda)

        buttonSalir.setOnClickListener{
            finish()
        }

        buttonCalculadora.setOnClickListener {
            val intent = Intent(this@MainActivity, CalculadoraActivity::class.java)
            startActivity(intent)
        }

        buttonGrafico.setOnClickListener {
            val intent = Intent(this@MainActivity, GraficoActivity::class.java)
            startActivity(intent)
        }

        buttonCrud.setOnClickListener {
            val intent = Intent(this@MainActivity, CrudActivity::class.java)
            startActivity(intent)
        }

        buttonCalculadoraBasica.setOnClickListener {
            val intent = Intent(this@MainActivity, CalculadoraBasicaActivity::class.java)
            startActivity(intent)
        }
        buttonWSTienda.setOnClickListener {
            val intent = Intent(this@MainActivity, WSTiendaActivity::class.java)
            startActivity(intent)
        }
    }
}